src/engine/Settings.cs

```diff
@@ -174,6 +176,14 @@ private Settings()
     /// </summary>
     public SettingValue<bool> CheatsEnabled { get; set; } = new SettingValue<bool>(false);
 
+    /// <summary>
+    ///   The current control scheme of the game.
+    ///   Key = InputMap name.
+    ///   Value = The events associated with the control.
+    /// </summary>
+    public SettingValue<Dictionary<string, List<InputEventWithModifiers>>> CurrentControlScheme { get; set; } =
```

Comment:
I don't think the control scheme needs saving, even if we add one. Control schemes should just be a list of presets that override the currently configured keys.

Reply:
After reading everything, turns out these are the current controls?
I think this should be renamed and if `Dictionary<string, List<InputEventWithModifiers>>` is changed to be a class, this'll be cleaned up pretty nicely.